### DISCUSSION TITLE
Great live spectrogram, bird identification not working

### DISCUSSION THREAD

**Discussion by Yourinoire** [mcguirepr89/BirdNET-Pi#1154] [Q&A]
Hi all,
new to github and a relative noob to coding. I made the Birdnet-work on the rpi-5 using instructions of https://github.com/pddpauw/BirdPi. Overnight it has been recording fine and identified some birds, great news so far. However, some issues:
1) since dawn chorus started, no single identification has been done two days in a row. Is this because it can be too much bird song? I believe this can not be the care hours in a row since I see some really clean spectrograms. It appears birdnet pi has stopped analyzing or something 
![Screenshot_1](https://github.com/mcguirepr89/BirdNET-Pi/assets/162788902/2f932cbc-081b-46d1-82b2-d290ef0aeab4)
![Screenshot_2](https://github.com/mcguirepr89/BirdNET-Pi/assets/162788902/ee143aeb-1794-4c74-96eb-ae003f59317b)

2) Where can I find a more detailed instruction on how to optimize settings and advanced setting like gain, compression, Birdnet Lite settings etc. ? A stepped approach to optimizing detection performance would be helpfiul and is probably available somewhere.
3) I tried to set notification to discord using the apprise notification configuration. the test works, but the actual identified birds do not get sent. Any suggestions?

Thanks so much all!

**Comment by MueJosh** [comment]
I don't have experience regarding your pi 5 

2. You should definitely check out [#39](https://github.com/mcguirepr89/BirdNET-Pi/discussions/39) .

Terminal command for gain etc.:
```
alsamixer -m
```
(You have to choose the usb sound card) 

Basically everything is answered at the [wiki](https://github.com/mcguirepr89/BirdNET-Pi/wiki)

3. Did you safe the settings? (I don't have another idea)
